tools/src/main/java/org/apache/kafka/tools/TopicCommand.java

```diff
@@ -50,7 +50,7 @@
 import org.apache.kafka.server.common.AdminOperationException;
 import org.apache.kafka.server.util.CommandDefaultOptions;
 import org.apache.kafka.server.util.CommandLineUtils;
-import org.apache.kafka.server.util.TopicFilter.IncludeList;
+import org.apache.kafka.tools.filter.TopicFilter.IncludeList;
```

Comment:
It seems there's a small build error.
```
> The following files had format violations:
      src/main/java/org/apache/kafka/tools/TopicCommand.java
          @@ -50,8 +50,8 @@
           import·org.apache.kafka.server.common.AdminOperationException;
           import·org.apache.kafka.server.util.CommandDefaultOptions;
           import·org.apache.kafka.server.util.CommandLineUtils;
          +import·org.apache.kafka.storage.internals.log.LogConfig;
           import·org.apache.kafka.tools.filter.TopicFilter.IncludeList;
          -import·org.apache.kafka.storage.internals.log.LogConfig;
           
           import·org.slf4j.Logger;
           import·org.slf4j.LoggerFactory;
  Run './gradlew :tools:spotlessApply' to fix these violations.
```

Reply:
I will push a PR to hot fix it.

Reply:
@chia7712, This hot fix PR in [here](https://github.com/apache/kafka/pull/17401), PTAL.

Reply:
Hey, thanks for quickly jumping on this and fixing the import order. Much appreciated.

Reply:
Not sure what happened. I'm pretty sure the PR had the "All checks have passed" mark before I merged it. Anyway thanks for the quick fix.

Reply:
> Not sure what happened. I'm pretty sure the PR had the "All checks have passed" mark before I merged it. Anyway thanks for the quick fix.

oh, out spotless check does not run with JDK 21. I open https://issues.apache.org/jira/browse/KAFKA-17718 to fix it

Reply:
Yeah, it would be good to understand what happened. Maybe the checkstyle didn't run for some reason? cc @mumrah

Reply:
> Yeah, it would be good to understand what happened. Maybe the checkstyle didn't run for some reason?

see https://issues.apache.org/jira/browse/KAFKA-17718 for root cause

Reply:
Thanks for checking and finding the issue!